modules/earthengine-layers/src/earth-engine-layer.js

```diff
@@ -8,22 +8,17 @@ import ee from '@google/earthengine';
 import {loadImageBitmap} from './image-utils/image-utils';
 // import {createMeshGrid} from './image-utils/image-utils';
 
-import SphericalMercator from '@mapbox/sphericalmercator';
-
-const merc = new SphericalMercator({size: 256});
-
 const eeApi = new EEApi();
 // Global access token, to allow single EE API initialization if using multiple
 // layers
 let accessToken;
 
 const defaultProps = {
-  /*
-  data: object,
-  token: string,
-  eeObject: String || object,
-  visParams: object
-  */
+  ...TileLayer.defaultProps,
+  data: {type: 'object', value: null},
```

Comment:
Maybe add comment saying it is unused? 

I guess we still need to decide whether we can reuse the data prop - maybe do some testing?

I think no other layer avoids using the `data` prop so this would be the first exception...

Reply:
But a `data` prop is required for a layer, right?